esrally/rally.py

```diff
@@ -859,6 +870,9 @@ def main():
             console.println("--target-hosts and --client-options must define the same keys for multi cluster setups.")
             exit(1)
     # split by component?
+    if sub_command == "download":
```

Comment:
Don't we need to support `--target-arch` and `--target-os` also for the the `install` subcommand?

Reply:
I would like to restrict this to the `download` subcommand for the moment because we should not support installing an artifact for a different platform IMHO as the successive startup is likely to fail in some scenarios and I also don't see why one would use this approach.